tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -41,6 +41,17 @@ int InternalCommandsHandler::execute(uint16_t clientId,
                                      uint32_t &outActualReplySize,
                                      uint32_t &outActualReplicaSpecificInfoSize,
                                      concordUtils::SpanWrapper &span) {
+  bool wedgeCommand = flags & 0x9;
+  if (wedgeCommand) {
+    LOG_INFO(m_logger, "A wedge command has been called seqNum: " << sequenceNum);
```

Comment:
Please use the KVLOG wrapper for sequenceNum so this log message looks the same as others. I realize this is just test output, but it's good to get into the habit.

Reply:
done